# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Free Software Foundation, Inc.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2000-03-24 11:13+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: ENCODING\n"

#: Pyblio/Autoload.py:57
#, c-format
msgid "warning: can't import %s: %s"
msgstr ""

#: Pyblio/Fields.py:265
msgid "Illegal year value"
msgstr ""

#: Pyblio/Fields.py:269
msgid "Illegal month value"
msgstr ""

#: Pyblio/Fields.py:273
msgid "Illegal day value"
msgstr ""

#: Pyblio/Fields.py:438
#, c-format
msgid "in %s"
msgstr ""

#: Pyblio/LyX.py:39
#, c-format
msgid "no input pipe `%s'"
msgstr ""

#: Pyblio/LyX.py:45
#, c-format
msgid "no output pipe `%s'"
msgstr ""

#: Pyblio/Open.py:97
#, c-format
msgid "method `%s' provides no opener"
msgstr ""

#: Pyblio/Open.py:105 Pyblio/Open.py:151
#, c-format
msgid "File `%s' does not exist"
msgstr ""

#: Pyblio/Open.py:116 Pyblio/Open.py:162
#, c-format
msgid "don't know how to open `%s'"
msgstr ""

#: Pyblio/Open.py:143
#, c-format
msgid "method `%s' provides no iterator"
msgstr ""

#: Pyblio/Format/BibTeX.py:321 Pyblio/Format/BibTeX.py:370
#, c-format
msgid "key `%s' is malformed"
msgstr ""

#: Pyblio/Format/BibTeX.py:345
#, c-format
msgid "%s:%d: key `%s' already defined"
msgstr ""

#: Pyblio/GnomeUI/Format.py:37
msgid "Format entries"
msgstr ""

#: Pyblio/GnomeUI/Format.py:51
msgid "Bibliography style:"
msgstr ""

#: Pyblio/GnomeUI/Format.py:52
msgid "Select style"
msgstr ""

#: Pyblio/GnomeUI/Format.py:58
msgid "Output format:"
msgstr ""

#: Pyblio/GnomeUI/Format.py:75
msgid "Output File:"
msgstr ""

#: Pyblio/GnomeUI/Format.py:76
msgid "Select output file"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:241
msgid "Day"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:250
msgid "Month"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:259
msgid "Year"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:284
msgid "Invalid day field in date"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:292
msgid "Invalid month field in date"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:300
msgid "Invalid year field in date"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:407
msgid "Entry type"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:409
msgid "Key"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:448
msgid "Create Field"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:499
msgid "Mandatory"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:499
msgid "Optional"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:499
msgid "Extra"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:543
msgid "New text"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:557
msgid "Invalid key format"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:565
#, c-format
msgid "Key `%s' already exists"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:632
msgid "Error in native string parsing"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:656 Pyblio/GnomeUI/Editor.py:693
msgid "Native Editing"
msgstr ""

#: Pyblio/GnomeUI/Editor.py:707
msgid "Standard Editing"
msgstr ""

#: Pyblio/GnomeUI/Document.py:89 Pyblio/GnomeUI/Document.py:165
#: Pyblio/GnomeUI/Document.py:177 Pyblio/GnomeUI/FileSelector.py:41
msgid "File"
msgstr ""

#: Pyblio/GnomeUI/FileSelector.py:71
msgid "Bibliography type:"
msgstr ""

#: Pyblio/GnomeUI/Document.py:75 Pyblio/GnomeUI/Index.py:64
msgid "Add..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:76 Pyblio/GnomeUI/Index.py:65
msgid "Edit..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:77 Pyblio/GnomeUI/Index.py:66
#: Pyblio/GnomeUI/Search.py:175
msgid "Delete"
msgstr ""

#: Pyblio/GnomeUI/Search.py:109
msgid "Search"
msgstr ""

#: Pyblio/GnomeUI/Search.py:132
msgid "Field"
msgstr ""

#: Pyblio/GnomeUI/Search.py:133
msgid "Pattern"
msgstr ""

#: Pyblio/GnomeUI/Search.py:148
msgid "Simple Search"
msgstr ""

#: Pyblio/GnomeUI/Search.py:152
msgid "Search command:"
msgstr ""

#: Pyblio/GnomeUI/Search.py:158
msgid "Expert Search"
msgstr ""

#: Pyblio/GnomeUI/Search.py:193
msgid "Full database"
msgstr ""

#: Pyblio/GnomeUI/Search.py:223
msgid "internal error during evaluation"
msgstr ""

#: Pyblio/GnomeUI/Search.py:272
#, c-format
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:41
msgid "Select sort criterions"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:52
msgid "In use"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:52
msgid "Sort criterions"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:63
msgid "[Entry Type]"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:64
msgid "[Key Value]"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:88
msgid "Reorder"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:91
msgid "Unselect all"
msgstr ""

#: Pyblio/GnomeUI/Sort.py:106 Pyblio/GnomeUI/Sort.py:128
msgid "Yes"
msgstr ""

#: Pyblio/GnomeUI/Utils.py:67
msgid ""
"The following errors occured:\n"
"\n"
msgstr ""

#: Pyblio/GnomeUI/Document.py:49
msgid "New"
msgstr ""

#: Pyblio/GnomeUI/Document.py:51
msgid "Merge..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:55 Pyblio/GnomeUI/Document.py:165
#: Pyblio/GnomeUI/Document.py:177
msgid "Previous Documents"
msgstr ""

#: Pyblio/GnomeUI/Document.py:62
msgid "About..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:63
msgid "Submit a Bug Report"
msgstr ""

#: Pyblio/GnomeUI/Document.py:80
msgid "Sort..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:84 Pyblio/GnomeUI/Document.py:91
#: Pyblio/GnomeUI/Document.py:100
msgid "Cite"
msgstr ""

#: Pyblio/GnomeUI/Document.py:85
msgid "Format..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:90
msgid "Edit"
msgstr ""

#: Pyblio/GnomeUI/Document.py:92
msgid "Help"
msgstr ""

#: Pyblio/GnomeUI/Document.py:96
msgid "Open"
msgstr ""

#: Pyblio/GnomeUI/Document.py:97
msgid "Save"
msgstr ""

#: Pyblio/GnomeUI/Document.py:99
msgid "Find"
msgstr ""

#: Pyblio/GnomeUI/Document.py:102
msgid "Close"
msgstr ""

#: Pyblio/GnomeUI/Document.py:198
#, c-format
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""

#: Pyblio/GnomeUI/Document.py:227
msgid "New database"
msgstr ""

#: Pyblio/GnomeUI/Document.py:235 Pyblio/GnomeUI/Document.py:239
msgid "[no entry]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:236
msgid "[1 entry]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:237
#, c-format
msgid "[%d entries]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:240
#, c-format
msgid "[%d/1 entry]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:241
#, c-format
msgid "[%d/%d entries]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:246
msgid "[modified]"
msgstr ""

#: Pyblio/GnomeUI/Document.py:256
msgid ""
"The database has been modified.\n"
"Discard changes ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:272
msgid "Merge file"
msgstr ""

#: Pyblio/GnomeUI/Document.py:284 Pyblio/GnomeUI/Document.py:339
msgid "Open error"
msgstr ""

#: Pyblio/GnomeUI/Document.py:308
msgid "Merge status"
msgstr ""

#: Pyblio/GnomeUI/Document.py:319 Pyblio/GnomeUI/Document.py:379
msgid "Open file"
msgstr ""

#: Pyblio/GnomeUI/Document.py:363
#, c-format
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""

#: Pyblio/GnomeUI/Document.py:368
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""

#: Pyblio/GnomeUI/Document.py:385
#, c-format
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:392
#, c-format
msgid ""
"During opening:\n"
"%s"
msgstr ""

#: Pyblio/GnomeUI/Document.py:427
#, c-format
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:458
msgid "Really remove all the entries ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:492
#, c-format
msgid "Really edit %d entries ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:526
#, c-format
msgid "Remove all the %d entries ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:528
#, c-format
msgid "Remove entry `%s' ?"
msgstr ""

#: Pyblio/GnomeUI/Document.py:586 Pyblio/GnomeUI/Document.py:593
#, c-format
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""

#: Pyblio/GnomeUI/Document.py:634
msgid ""
"Please install bug-buddy\n"
"to use this feature"
msgstr ""

#: Pyblio/GnomeUI/Document.py:645
msgid "This program is copyrighted under the GNU GPL"
msgstr ""

#: Pyblio/GnomeUI/Document.py:647
msgid "Gnome interface to the Pybliographer system."
msgstr ""

#: Pyblio/GnomeUI/Document.py:651
msgid "Pybliographer Home Page"
msgstr ""

#: pybliographer.py:38
#, c-format
msgid "This is %s, version %s"
msgstr ""

#: pybliographer.py:42 pybliographer.py:49
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr ""

#: pybliographer.py:43
msgid "For details, type `warranty'."
msgstr ""

#: pybliographer.py:108 pybliographer.py:157
#, c-format
msgid "%s: error: can't open file `%s'"
msgstr ""

#: pybliographer.py:121
#, c-format
msgid "For help, run %s and type `help' at the prompt"
msgstr ""

#: pybliographer.py:130
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""

#: scripts/pybcompact.py:30
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr ""

#: scripts/pybcompact.py:34
#, c-format
msgid "pybcompact: error: %s\n"
msgstr ""

#: scripts/pybcompact.py:97
msgid "no entry"
msgstr ""

#: scripts/pybcompact.py:142
#, c-format
msgid "can't find the following entries: %s"
msgstr ""

#: scripts/pybconvert.py:27
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr ""

#: scripts/pybconvert.py:36
msgid "pybconvert: bad conversion format"
msgstr ""

#: scripts/pybformat.py:34
msgid "usage: pybformat [bibtexfiles...]\n"
msgstr ""

#: scripts/pybformat.py:38
#, c-format
msgid "pybformat: error: %s\n"
msgstr ""

#: scripts/pybformat.py:44
#, c-format
msgid "pybformat: warning: %s\n"
msgstr ""

#: scripts/pybformat.py:140 scripts/pybtex.py:84
#, c-format
msgid "can't find style `%s'"
msgstr ""

#: scripts/pybtex.py:34
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr ""

#: scripts/pybtex.py:38
#, c-format
msgid "pybtex: error: %s\n"
msgstr ""

#: scripts/pybtex.py:59
msgid "pybtex: warning: the following keys were not resolved"
msgstr ""

#: scripts/pybtex.py:64
msgid "no style defined"
msgstr ""
